clients/go/apis/v1alpha1/config_test.go

```diff
@@ -149,180 +152,226 @@ func TestValidate(t *testing.T) {
 		name    string
 		cfg     *Config
 		wantErr string
-	}{{
```

Comment:
Please try to keep the current bracelets arrangement to avoid causing unnecessary changes appearing.

Reply:
The format of this files was not standard, therefore I changed it. I didn't do it manaully, I ran the code formatter in IDE. Therefore, I believe no business logic has been changed.

Reply:
I don't think standard go formatter will change this. Please adjust your IDE so future PRs won't have such unintended changes. Thanks.

Reply:
For example, you can see the format of this file is different from the format of  file https://github.com/abcxyz/lumberjack/blob/main/clients/go/apis/v1alpha1/auditlogutil_test.go

Reply:
The linter should have caught this change when it was submitted. It's possible these changes came in _before_ the linter was connected.

Reply:
@capri-xiyue That's not what I meant. Both styles of bracelet are technically acceptable by gofmt. I don't see a need to force converge to one style.

@sethvargo Are you sure linter will report this as an issue?

Reply:
Nope :)